cypress/pages/Registpage.js

```diff
@@ -0,0 +1,153 @@
+export const Registpage = {
+    TXT_FIRSTNAME: "#firstName",
+    TXT_LASTNAME: "#lastName",
+    TXT_EMAIL: "#userEmail",
+    RDO_GENDER: "#gender-radio-2",
```

Comment:
For locators, they are not wrong, just not flexible. Due to your work, if I want to select different gender, DoB and hobbies, I have to replace the locators or define new ones